airship.cabal

```diff
@@ -47,6 +48,7 @@ library
                       , http-media
                       , http-types == 0.8.*
                       , lifted-base == 0.2.*
+                      , microlens
```

Comment:
A good question. I figure that since we only use a _tiny_ subset of the operators microlens provides (literally just `Lens` and `^.`, which probably won't change), specifying no bounds is acceptable here. If we used microlens more extensively, I'd provide `maj.min.*`, but here I think it's fine.